swagger.yaml

```diff
@@ -0,0 +1,211 @@
+swagger: "2.0"
+info:
+  description: "An API used to query codes for information"
+  version: "1.0.0"
+  title: "ONS Code List API"
+  license:
+    name: "Open Government Licence v3.0"
+    url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+basePath: "/v1"
+tags:
+- name: "Code List"
+schemes:
+- "http"
+parameters:
+  id:
+    name: id
+    description: "Id for a list code"
+    in: path
+    type: string
+    required: true
+  codeId:
+    name: codeId
+    description: "CodeId for a discrete value within a code list"
+    in: path
+    type: string
+    required: true
+  pageSize:
```

Comment:
To be able to retrieve a set of data within a list, it would be much more flexible to have `maxResults` then `pageSize` and then `offset` variable to replace `pageNumber`. As discussed with other specs (e.g. dataSet spec).